src/Lean/Elab/Quotation.lean

```diff
@@ -194,7 +194,6 @@ def addNamedQuotInfo (stx : Syntax) (k : SyntaxNodeKind) : TermElabM SyntaxNodeK
     if s.length > 3 then
       if let (some l, some r) := (stx[0].getPos? true, stx[0].getTailPos? true) then
         -- HACK: The atom is the string "`(foo|", so chop off the edges.
-        -- HACK: We have to use .original here or the hover won't show up
```

Comment:
The code hasn't changed here, but it's no longer considered a hack but a supported use case now.